### ISSUE TITLE
Few Questions

### ISSUE THREAD

**Issue by ConstantinosM** [RajDandekar/MIT-Global-COVID-Modelling-Project-1#1]
Thanks a lot for publishing the code of this very interesting work!I have a couple of questions if you could answer.

- I am fairly new to Julia and I was wondering if you could give some indicators of how long does the training (1 step) take to converge in e.g. Italy. I am aware that in countries with low Recovered you are running a 2-step optimization scheme but I would like to have a sense of time of convergence.

- For the loss function, I see that you are using absolute value on the Infected count. Is there a reason for this?

- Did you encounter instability issues (underflow in integration). I am trying to replicate this in Tensorflow with `tfdiffeq` library.

(This is not an actual issue but I couldn't find any other way to ask you :) )

**Comment by RajDandekar** [comment]
Hello! Sorry for the late answer.

(a) It takes around 40-50 mins for 60000 iterations. 

(b) If the infected count takes negative value at any point during the training process, I have used absolute since a log function is used in the loss.

(c) I've not encountered instability issues (underflow in integration for example). Would be interested to know if you face any replicating in TensorFlow

**Comment by Luanyunqiang** [comment]
Hi, Thanks for the code, I'm very interested in this program and have few questions if you could answer.

1. For some countries like Brazil, there is a stage 2 iterations. Is that means you didn't train the γ and δ parameters in stage 2 and use the trained value in stage 1? 
2. And for stage 2 iterations, you only use the loss function that only with the infected?
